### DISCUSSION TITLE
Welcome to LLaVA Discussions!

### DISCUSSION THREAD

**Discussion by haotian-liu** [haotian-liu/LLaVA#41] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mudomau** [comment]
> ## wave Welcome!
> 
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
>     * Ask questions you’re wondering about.
> 
>     * Share ideas.
> 
>     * Engage with other community members.
> 
>     * Welcome others and are open-minded. Remember that this is a community we
>       build together muscle.
> 
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Hi! I'm mau, just a guy whos curious about AI. I started experimenting with LLaMa/Alpaca/Vicuna when they came out, and decided to try now LLaVA too :smiley: Really impressed with it so far. Looking forward to seeing how your project develops.

https://user-images.githubusercontent.com/2564029/234418909-3f74ae37-5eb6-4abb-894d-48625f466366.mp4

**Comment by zolf1983** [comment]
执行以下语句：python -m llava.serve.model_worker --host 192.168.2.114 --controller http://192.168.2.114:10000/ --port 40000 --worker http://192.168.2.114:40000/ --model-path E:/llava/llava-v1.5-13b 
报错: OS错误: E:/llava/llava-v1.5-13b 似乎没有名为 preprocessor_config.json的文件.查看可用的文件的"[https://huggingface.co/E:/llava/llava-v1.5-13b/main"](https://huggingface.co/E:/llava/llava-v1.5-13b/main).想启动本地的模型为什么会报这样的错误呢？启动本地模型需要改什么配置呢？
本地模型文件在https://huggingface.co/liuhaotian/llava-v1.5-13b/tree/main 下载的，放在本地E:/llava/llava-v1.5-13b文件夹下。

**Comment by HarshmJenya** [comment]
I have tried to load the model running the command:
 python -m llava.serve.model_worker --host "0.0.0.0" --controller-address "http://localhost:10000" --port 40000 --worker-address "http://localhost:40000" --model-path "models/llava-v1.5-7b" --load-8bit 
and 
encountered the following error 

![image](https://github.com/haotian-liu/LLaVA/assets/142071812/1180ea23-f10c-4494-b259-342b22967842)

**Comment by Marco-DAmato** [comment]
Hi,

Is LLaVA 1.5 or even better 1.6 preview available in Google Colab? I mean the entire pipeline where the model is trained and fine-tuned in Google Colab. Anyone that can help?

Thanks for your time.

**Comment by Eilyn** [comment]
Hello, I am trying to run the liuhaotian/llava-v1.5-13b sample, however I am getting the CUDA out of memory error. I saw in the read me that there are some parameters which could be used to run the code with less GPUs, however I cannot find where to configure those.

![image](https://github.com/haotian-liu/LLaVA/assets/10180103/4dcf1d36-3a6e-4e41-a4c8-7b805d35f7e5)

Any advice?

**Comment by kalle07** [comment]
i tryed llava 1.5 and 1.6 its not better in recognizing images than boro-tag-coco.
Who has other experiences?

**Comment by WwangYiheng** [comment]
大佬你好，我在服务器上包llava-v1.5-7b的时候报ValueError: Unrecognized configuration class <class 'transformers.models.clip.configuration_clip.CLIPConfig'> for this kind of AutoModel: AutoModelForCausalLM.
Model type should be one of BartConfig, BertConfig, BertGenerationConfig, BigBirdConfig, BigBirdPegasusConfig, BioGptConfig, BlenderbotConfig, BlenderbotSmallConfig, BloomConfig, CamembertConfig, LlamaConfig, CodeGenConfig, CpmAntConfig, CTRLConfig, Data2VecTextConfig, ElectraConfig, ErnieConfig, FalconConfig, FuyuConfig, GitConfig, GPT2Config, GPT2Config, GPTBigCodeConfig, GPTNeoConfig, GPTNeoXConfig, GPTNeoXJapaneseConfig, GPTJConfig, LlamaConfig, MarianConfig, MBartConfig, MegaConfig, MegatronBertConfig, MistralConfig, MixtralConfig, MptConfig, MusicgenConfig, MvpConfig, OpenLlamaConfig, OpenAIGPTConfig, OPTConfig, PegasusConfig, PersimmonConfig, PhiConfig, PLBartConfig, ProphetNetConfig, QDQBertConfig, ReformerConfig, RemBertConfig, RobertaConfig, RobertaPreLayerNormConfig, RoCBertConfig, RoFormerConfig, RwkvConfig, Speech2Text2Config, TransfoXLConfig, TrOCRConfig, WhisperConfig, XGLMConfig, XLMConfig, XLMProphetNetConfig, XLMRobertaConfig, XLMRobertaXLConfig, XLNetConfig, XmodConfig, LlavaConfig, LlavaMptConfig, LlavaMistralConfig.错误，命令是这个python -m llava.serve.cli     --model-path /root/autodl-tmp/LLaVA/clip-vit-large-patch14-336     --image-file /root/autodl-tmp/LLaVA/images/llava_v1_5_radar.jpg     --load-4bit
